discovery/roles/postscripts/common/tasks/configure_postscripts.yml

```diff
@@ -17,6 +17,12 @@
   ansible.builtin.command: "{{ xcat_path }}/chdef all postscripts=\"syslog,remoteshell,syncfiles\""
   changed_when: true
 
+- name: Apply role postscripts to xCAT groups
+  ansible.builtin.command:
+    cmd: "{{ xcat_path }}/chdef {{ item }} -p postscripts=omnia_{{ item }}"
+  loop: {{ fetch_unique_roles | list | difference(['default']) }}
```

Comment:
we need to have condition if that role does not have any nodes also. add rescue for that